libkpod/container_data.go

```diff
@@ -253,7 +252,7 @@ func getOCIContainer(store storage.Store, container string) (*oci.Container, err
 	return oci.NewContainer(ctr.ID, name, containerPath, m.Annotations[annotations.LogPath], nil, labels, kubeAnnotations, img, &metadata, ctr.ImageID, tty, stdin, stdinOnce, false, false, containerDir, created, m.Annotations["org.opencontainers.image.stopSignal"])
 }
 
-func getOCIRuntime(store storage.Store, container string) (*oci.Runtime, error) {
+func (c *ContainerServer) getOCIRuntime(container string) (*oci.Runtime, error) {
```

Comment:
You're right.  In fact, it might as well be removed entirely.  ContainerServer.Runtime() can be called instead